code/ietf-sid-file.yang

```diff
@@ -235,6 +256,32 @@ module ietf-sid-file {
         the YANG module identified by 'module-name' and
         'module-revision'.";
 
+      leaf status {
+        description
+          "The status field contains either the module-revision at
+          which this SID item was permanently allocated, or the
+          string 'unstable'.";
+        type enumeration {
+          enum stable {
+            description "This SID allocation has been published in a
+                         stable catalog.";
+            value 0;
```

Comment:
the term catalog is not really used anywhere.

This is an important distinction.
The status refers to the SID file published in a catalog of some sort
and not the status of the YANG module that the SID file is derived from.

Not sure about the unintended consequences of SID item status that
was not algorithmically derived from the YANG module.

Reply:
catalog: fixed in dc18984

An important distinction, indeed (as with the sid-file-status, which may be unpublished for a published module!).